src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,35 @@
+.card__rating {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 16px;
+  width: 100%;
+  gap: 4px;
+
+  .stars {
+    display: flex;
+    gap: 4px;
+
+    &__star {
+      background-image: url(../images/star.svg);
+      width: 16px;
+      height: 16px;
+      background-position: center;
+      background-repeat: no-repeat;
+    }
+
+    &--4 {
+      .stars__star:nth-child(-n + 4) {
+        background-image: url(../images/star-active.svg);
+      }
+    }
```

Comment:
The implementation of the `stars--N` modifiers is not dynamic. Ensure that the star rating system can dynamically adjust based on different ratings as required by the task.